src/DataFixtures/UserFixtures.php

```diff
@@ -26,19 +26,23 @@ public function load(ObjectManager $manager)
         $users = [
             [
                 'username' => 'Mathias',
-                'email' => 'mathias.micheli@mail.com'
+                'email' => 'mathias.micheli@mail.com',
+                'role' => ['ROLE_ADMIN']
```

Comment:
```suggestion
                'roles' => ['ROLE_ADMIN']
```

A ajouter sur les autres précédents